docs/source/camera.rst

```diff
@@ -5,11 +5,26 @@ Camera
 Setting
 =============
 
-One can set the direction and zoom of the camera:
+Camera has three settings:
+
+- zoom: the zoom level of the camera
+- position: the position of the camera
+- look_at: the point the camera is looking at
+
+.. code-block:: python
+
+    viewer.camera.zoom = 2
+    viewer.camera.position = [0, 0, 100]
+    viewer.camera.look_at = [0, 0, 0]
+
+
+If you want to set the `direction` of the camera, you can use the following code:
 
 .. code-block:: python
 
-    viewer.cameraSetting = {"direction": [0, 5, 1], "zoom": 2}
+    # the look_at is the center of the atoms (or the center of the bounding box in case of no atoms)
+    # the distance is the distance between the camera and the look_at point
+    viewer.camera.setting = {"direction": [0, 5, 1], "distance": 50, "zoom": 2}
```

Comment:
is setting different from the top-level settings or just another way of providing settings?

if it's just another way, do we really need both?

also the `direction`, `distance`, etc. seem to disagree with the claim that the camera has just 3 settings

Reply:
I update the doc in [053e16b](https://github.com/superstar54/weas-widget/pull/47/commits/053e16b66c1669819c2725b5b205aabc7444b906); hope this makes it clear.